src/modules/paymentProcessor/bridging/ConnextBridgeLogic.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+import {CrosschainBase_v1} from
+    "src/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/interfaces/IERC20PaymentClientBase_v1.sol";
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+
+interface IEverclearSpoke {
+    function newIntent(
+        uint32[] memory destinations,
+        address to,
+        address inputAsset,
+        address outputAsset,
+        uint amount,
+        uint24 maxFee,
+        uint48 ttl,
```

Comment:
What does ttl stand for :D

Reply:
Time to live aka a deadline check for a solver to fill an intent. This helps avoid slippage attacks